tokio-test/src/macros.rs

```diff
@@ -0,0 +1,41 @@
+//! A collection of useful macros for testing futures and tokio based code
+
+/// Assert if a future is ready
+#[macro_export]
+macro_rules! assert_ready {
+    ($e:expr) => {{
+        match $e {
+            Ok(::futures::Async::Ready(v)) => v,
+            Ok(_) => panic!("not ready"),
```

Comment:
It might be useful if we had a 
```suggestion
            Ok(_) => panic!(concat!(stringify($e), ": not ready")),
```
or something here to make debugging these panics easier?

Reply:
Also, I notice that `tokio-timer`'s test support code has a similar macro that allows adding an optional message to be printed when the assertion fails (like `std::assert`)":
https://github.com/tokio-rs/tokio/blob/4271a9cd8dc8d43249f5a4febe5fea19f4dca971/tokio-timer/tests/support/mod.rs#L23-L34

Can we add something like that to this macro?